### PR TITLE
[@tools-dapp] Add right sidebar with extra info

### PR DESCRIPTION
**Pull request by KristinaSpasevska**

### CODE REVIEW

File: packages/apps/tools/src/pages/transactions/transfer/index.tsx [hunk 1]
```diff
@@ -38,6 +39,28 @@ const TransferPage = () => {
   const [receiverChainId, setReceiverChainId] = useState<ChainId>(CHAINS[0]);
   const [senderChainId, setSenderChainId] = useState<ChainId>(CHAINS[0]);
 
+  const [sidebarOpen, setSidebarOpen] = useState<boolean>(false);
+
+  const helpInfoSections = [
+    {
+      question: 'What was my key index, again?',
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/pages/transactions/transfer/index.tsx:46]: Should we have these coming from the translation file as well?

File: packages/apps/tools/src/components/Partials/transactions/transfer/right-info-sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+export const RightInfoSidebar: FC<IRightInfoSidebarProps> = ({
+  infoSections,
+  sidebarOpen,
+}) => {
+  const { t } = useTranslation('common');
+  const router = useRouter();
+  const isMediumScreen = useIsMatchingMediaQuery(`${breakpoints.lg}`);
+
+  console.log(isMediumScreen);
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Partials/transactions/transfer/right-info-sidebar.tsx:37]: Can be removed?

File: packages/apps/tools/src/components/Partials/transactions/transfer/right-info-sidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+  const [initialItem, setInitialItem] = useState<{ item: number } | undefined>(
+    undefined,
+  );
+
+  const drawerPanelRef = useRef<HTMLElement | null>(null);
+
+  useEffect(() => {
+    if (isMediumScreen) {
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Partials/transactions/transfer/right-info-sidebar.tsx:46]: We can not open the sidebar on mobile devices?

File: packages/apps/tools/src/pages/transactions/transfer/index.tsx (hunk 1 above)
Comment (KristinaSpasevska) [reply packages/apps/tools/src/pages/transactions/transfer/index.tsx:46]: I thought when we have the final ones, we would create a separate PR and add these in the translations files.

File: packages/apps/tools/src/components/Partials/transactions/transfer/right-info-sidebar.tsx (hunk 3 above)
Comment (KristinaSpasevska) [reply packages/apps/tools/src/components/Partials/transactions/transfer/right-info-sidebar.tsx:46]: We can, this is for the initial state

Comment (sanderlooijenga) [reply packages/apps/tools/src/components/Partials/transactions/transfer/right-info-sidebar.tsx:46]: I couldn't open it when clicking the question mark icon on mobile view

File: packages/apps/tools/src/pages/transactions/transfer/index.tsx (hunk 1 above)
Comment (sanderlooijenga) [reply packages/apps/tools/src/pages/transactions/transfer/index.tsx:46]: Makes sense 👍

File: packages/apps/tools/src/components/Partials/transactions/transfer/right-info-sidebar.tsx (hunk 3 above)
Comment (KristinaSpasevska) [reply packages/apps/tools/src/components/Partials/transactions/transfer/right-info-sidebar.tsx]: this is changed, check last commit

**Review by sanderlooijenga — APPROVED** [review]

**Pull request merged**